Sources/Benchmark/BenchmarkReporter.swift

```diff
@@ -69,8 +80,13 @@ struct PlainTextReporter: BenchmarkReporter {
         for index in 0...results.count {
             for columnIndex in 0..<columns.count {
                 let cell = columns[columnIndex][index]
-                let padded = cell.padding(
+                var padded = ""
+                if (index != 0 && columnIndex == 1) {
+                    padded = cell.leftPadding(toLength: widths[columnIndex], withPad:" ")
+                } else {
+                    padded = cell.padding(
                     toLength: widths[columnIndex], withPad: " ", startingAt: 0)
+                }
```

Comment:
Hmm, given that we don't have tests for this already, I won't hold you to this standard, but if you would like to go for the extra credit, setting up some simple tests for our console output would be awesome!

Reply:
https://github.com/google/swift-benchmark/blob/0eee05131124a31c632502e3e6f761402807b6c5/Tests/BenchmarkTests/BenchmarkReporterTests.swift#L28
I added the test that confirms the format(padded right or padded left) for console output.